script/setup.ts

```diff
@@ -63,11 +69,13 @@ const clientStep: Task[] = [
     label: "Client build",
   },
 ];
-const serverBuild: Task[] = [
```

Comment:
@jainpawan21 Why did you separate the task to separated arrays?
The task is executing in a Promise.all context

Reply:
I separated them because these two tasks were not executing and logging correctly. After separating these two, they execute in the right order.
In the below screen recording, 
It first logs **Executing server Build** but then instead of logging **Finished server Build**, It first logs **Finished prisma Generation** and then **Finished perver Build** and it never logs **Executing prisma generation**

https://user-images.githubusercontent.com/39362422/167293303-aaf7e3d5-e5b6-4d81-bfa4-07f259bdd5e2.mov

Reply:
Ok, got it